vignettes/bds_exposure.Rmd

```diff
@@ -1,8 +1,7 @@
 ---
 title: "Creating an Exposure (BDS) ADaM"
```

Comment:
Should we rename it to "Creating a BDS Exposure ADaM" to be in line with the other BDS vignettes?

Reply:
OK, updated